src/lib/components/composites/statistics-components/StageProgress.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+    import type { StageProgressInterface } from "$lib/model/component-interfaces";
+    import Circle from "lucide-svelte/icons/circle";
+    import ErrorIndicator from "$lib/components/composites/utils/ErrorIndicator.svelte";
+    import { getStatusColor } from "$lib/utils/common-helper";
+    import { type PaperStatus } from "$lib/model/general";
+    import StageProgressChart from "$lib/components/composites/statistics-components/StageProgressChart.svelte";
+    import { Skeleton } from "$lib/components/primitives/skeleton";
+
+    interface StageProgressChartProps {
+        stageProgress: Promise<StageProgressInterface>;
+    }
+
+    const { stageProgress }: StageProgressChartProps = $props();
+</script>
+
+<!--
+@component
+Displays the stage progress, i.e. the number of papers unreviewed, undecided, accepted and declined in the current stage.
+The stage progress is visualized using
+1) a donut chart
+2) and a corresponding legend (= list of decisions with their count)
+
+Usage:
+```svelte
+    <StageProgress
+        {stageProgress}
+    />
+```
+-->
+<div class="flex flex-row items-center gap-x-4 p-5" data-testid="stage-progress">
+    {#await stageProgress}
+        <Skeleton class="h-[150px] w-[150px] rounded-full" />
+        <div class="mx-10 flex flex-col gap-y-3">
+            {#each { length: 4 }}
+                <Skeleton class="flex h-6 min-w-[190px] rounded-full" />
+            {/each}
+        </div>
+    {:then { stage, decisions }}
+        <StageProgressChart {decisions} {stage} />
+        <ul class="mx-10 space-y-3">
+            {#each Object.entries(decisions) as [decision, number] (decision)}
+                <li class="flex flex-row items-center">
+                    <Circle
+                        class="mr-2 {getStatusColor(decision as PaperStatus)}"
+                        size={14}
+                        strokeWidth={4}
+                    />
+                    <span class="text-default-sb min-w-[150px]">{decision}:</span>
```

Comment:
Is it important that it's exactly 150px?
```suggestion
                    <span class="text-default-sb min-w-38">{decision}:</span>
```

Reply:
No same here, I only reused the widths etc. from the mockup, but I agree that it seems cleaner to use w-38. I will change it.